pkg/util/util.go

```diff
@@ -104,10 +106,17 @@ func Forever(f func(), period time.Duration) {
 }
 
 // Until loops until stop channel is closed, running f every period.
+// Until is syntactic sugar on top of JitterUntil with zero jitter factor
+func Until(f func(), period time.Duration, stopCh <-chan struct{}) {
+	JitterUntil(f, period, 0.0, stopCh)
+}
+
+// JitterUntil loops until stop channel is closed, running f every period.
+// If jitterFactor is positive, the period is jittered before every run of f.
```

Comment:
This should spell out the semantics for negative jitter factor.  TBH, negative jitter factor doesn't make a lot of sense to me -- is it valid?  What does it mean?